deploy/ray-operator/api/v1alpha1/raycluster_types.go

```diff
@@ -56,8 +56,8 @@ type RayClusterImage struct {
 }
 
 type Extension struct {
-	// the size of this pod group
-	NumNodes int32 `json:"numNodes"`
+	// the replicas of this pod group
+	Replicas int32 `json:"replicas"`
 
 	// worker/head
 	Type string `json:"type,omitempty"`
```

Comment:
If the type is a Enum, prefer to define a struct:

```go
type ReplicaType string

const (
  ReplicaTypeWorker ReplicaType = "worker"
  ReplicaTypeHead = "head"
)
```

Reply:
Sure~